.github/workflows/pr-visual-regression.yml

```diff
@@ -0,0 +1,60 @@
+# This workflow runs Percy visual regression tests for pull requests. It extracts the list of stories
+# from the PR body and runs snapshots for those stories to validate visual changes before merging.
+# If Percy detects visual differences, the workflow fails to block merging.
+
+name: Percy PR Visual Regression Tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+      - labeled
+
+jobs:
+  visual-regression:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checkout the repository
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # Set up Node.js
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 22
```

Comment:
Lets remove this, we should always be building on the LTS node.

Reply:
If we don't specify this, it always assumes it?